src/utils/string.rs

```diff
@@ -0,0 +1,69 @@
+use std::ops::{Range, RangeFrom, RangeFull, RangeTo};
+use itertools::Itertools;
+
+// This trait is already contained in the standard lib, however it is unstable.
+// When the feature will stabilize, pls remove!
```

Comment:
It may be a good idea to write this second line as a `TODO`, linking to the relevant RFC/PR in the Rust repo.

```
// TODO: Remove when the `collections_range` feature stabilises
// (https://github.com/rust-lang/rust/issues/30877)
```

Writing it that way tends to make it easier to `grep` for and will immediately pop out as something which needs to be done, instead of being just another comment.

Reply:
Makes sense. Thanks. Done.